File: Baekjoon/2920음계.cpp
#include<stdio.h>
#pragma warning(disable :4996)

int main() {
	int eum[8] = { 1, 2, 3, 4, 5, 6, 7, 8 };
	int inp[8];
	int asc = 0, des = 0;
	for(int i = 0; i < 8; i++)
		scanf("%d", &inp[i]);
	for (int i = 0; i < 8; i++) {
		if (eum[i] == inp[i])
			asc++;
		else if (eum[8 - (i + 1)] == inp[i])
			des++;
	}
	if(asc == 8)
		printf("ascending\n");
	else if (des == 8)
		printf("descending\n");
	else
		printf("mixed\n");
	
	return 0;
}

File: Baekjoon/2920음계.py
arr = list(map(int, input().split()))

isAscending = True
isDescending = True

for i in range(1, 8):
    if arr[i] > arr[i - 1]:
        isDescending = False
    elif arr[i] < arr[i - 1]:
        isAscending = False
    if isAscending == False and isDescending == False:
        break

if(isAscending):
    print("ascending")
elif(isDescending):
    print("descending")
else:
    print("mixed")